content/building-blocks/configure-your-computer/infrastructure-choice/configure-git.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Setting Up a New Workspace and Configuring Git "
+description: "Learn how to set up a new workspace/VM instance on Research Cloud and configure Git on the instance."
+keywords: "surf, rd, rclone, large files, webdav, store, storage, git, configure"
+weight: 3
+author: "Roshini Sudhaharan"
+authorlink: "https://nl.linkedin.com/in/roshinisudhaharan"
+draft: false
+aliases:
+  - /setup/research-drive
+---
+
+# Overview
+
+This building block takes you through the steps to configure Git on a new virtual machine (VM) on Research Cloud once you have finished [setting up your account](https://tilburgsciencehub.com/configure/research-cloud/?utm_campaign=referral-short). Follow [these steps](https://servicedesk.surf.nl/wiki/display/WIKI/Start+a+simple+workspace) to create a workspace and [log in](https://servicedesk.surf.nl/wiki/display/WIKI/Log+in+to+your+workspace) to the instance.
```

Comment:
Maybe put this link instead of the Lisa one? It has the basics of setting up research cloud, requesting access etc

https://tilburgsciencehub.com/building-blocks/configure-your-computer/infrastructure-choice/getting-started-research-cloud/